docker/run/tomato-ctl.py

```diff
@@ -330,7 +330,7 @@ def generate_default_config():
 			# enabled: True,
 			# is_database: True
 			'image': 'mongo:latest',
-			'ports': ('127.0.0.1:27017', 27017),
+			'ports': [27017],
```

Comment:
We should find a different way to connect to the database than exposing it to the world